docs/policy/data_management_plans.md

```diff
@@ -0,0 +1,122 @@
+# Data Management Plans
+
+OpenNeuro is a public resource intended to make neuroimaging research data available to as
+broad an audience as possible. This page is intended to help researchers and reviewers evaluate
+whether OpenNeuro meets your criteria for a data host.
+
+For further details, please consult our [](data_retention.md).
```

Comment:
```suggestion
For further details, please consult our [Data Retention Policies](data_retention.md).
```

Reply:
This renders with the title of the document.